iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+        labelMMIWInfo.text = NSLocalizedString("slideview1text", comment: "text")
+        
+        pc1View.backgroundColor = UIColor.red
+        pc2View.backgroundColor = UIColor.white
+        pc3View.backgroundColor = UIColor.white
+        pc4View.backgroundColor = UIColor.white
+
+        buttonNext.alpha = 1.0
+        buttonNext.setTitleColor(UIColor.white, for:.normal)
+        buttonNext.setTitleColor(UIColor.green, for:.selected)
+
+        buttonNext.imageEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.contentEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.titleLabel?.font = UIFont.systemFont(ofSize: 17)
+        buttonNext.setTitle("Next", for:.normal)
+        
+
+        buttonNext.addTarget(self, action: #selector(self.changePage(sender:)), for: .touchUpInside)
+        
+        let bottomView = UIView(frame:CGRect(x:0, y:0, width:100, height:50))
+        bottomView.addSubview(pc1View)
+        bottomView.addSubview(pc2View)
+        bottomView.addSubview(pc3View)
+        bottomView.addSubview(pc4View)
+        bottomView.addSubview(buttonNext)
+
+        let gr = UITapGestureRecognizer(target: self, action: #selector(self.changePage(sender:)))
+
+        bottomView.isUserInteractionEnabled = true
+        bottomView.addGestureRecognizer(gr)
+        
+        bottomView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(bottomView)
+        
+        
+        var views: [String:Any] = ["labelMMIW":labelMMIW, "labelMMIWInfo":labelMMIWInfo, "bottomView":bottomView]
+
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-36-[labelMMIW]-56-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views: ["labelMMIW":labelMMIW]))
+ 
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-25-[labelMMIWInfo]-25-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views: ["labelMMIWInfo":labelMMIWInfo]))
+
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-30-[labelMMIW(50)]->=0-[labelMMIWInfo(160)]-12-[bottomView(17)]-36-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views:views))
+
+        // page control and button constraints
+
+        self.view.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|->=0-[bottomView(70)]-10-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views:views))
+       
+        views = ["pc1View":pc1View, "pc2View":pc2View, "pc3View":pc3View, "pc4View":pc4View, "buttonNext":buttonNext]
+
+         // bottomView constraints
+        
+        bottomView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|->=0-[pc1View(15)]-2-|", options:NSLayoutConstraint.FormatOptions.init(rawValue:0), metrics:nil, views:views))
+        
+        /*let constraint = NSLayoutConstraint(
```

Comment:
This is what I did to center the first bar of the paging indicator as a test (we would need  make this same constraint for all bars and the Next button).  It did raise the bar, but a little too far up. Don't know if there is an easy way to increase Next button clickable area.